### PR TITLE
Add documentation for lb-check

### LINKED ISSUE #25077: Publish information about Infinispan availability in lb-check if MULTI_SITE is enabled
### Description

Introduce a new check available for the load balancer to decide whether this site should receive load or not.

This is Keycloak implementation part of the following issue for Keycloak benchmark:
https://github.com/keycloak/keycloak-benchmark/issues/591

### PR DESCRIPTION
**Pull request by mhajas**
Closes #25077

### CODE REVIEW

**Comment by ahus1** [comment]
@mhajas - I've added another commit to update the Keycloak deployment to contain the enabled feature `multi-site`. 

There is a PR in KCB https://github.com/keycloak/keycloak-benchmark/pull/609 - I want to wait until the next nightly build to give it a try before I merge this. If I merge it, I want to cherry-pick this merge commit to the two backport PRs.

Let me know what you think about it.

File: docs/guides/high-availability/examples/generated/ispn-single.yaml
Comment (mhajas) [inline docs/guides/high-availability/examples/generated/ispn-single.yaml:28]: @ahus1 We should not expose this in the production environment. This is needed only for our testing. I think we should include some production switch to our Helm chart. WDYT?

Comment (ahus1) [reply docs/guides/high-availability/examples/generated/ispn-single.yaml]: While it is generated in the big file, we never show this part in the docs. Feel free to go ahead and update the helm chart so we can update it in the generation. It might still be enabled by default for our purposes.

Comment (mhajas) [reply docs/guides/high-availability/examples/generated/ispn-single.yaml:28]: I see, that makes sense. I will update the helm chart and we can change this later. This is not blocker for merging then

**Review by ahus1 — APPROVED** [review]
Approving @mhajas's changes, and thank you @mhajas for having a look at my on-top changes.

**Comment by mhajas** [comment]
@ahus1 I forgot to write it here officially. Yes, I looked at your changes and they make sense to me! Thank you!

**Pull request merged**
